### PR TITLE
Fixed incorrect rate metric in StatsManager

### PR DESCRIPTION
**Pull request by dutor**
This bug seems due to improper support to customized clock by `folly::MultiLevelTimeSeries`

### CODE REVIEW

File: src/common/stats/StatsManager.inl [hunk 1]
```diff
@@ -19,9 +19,9 @@ StatsManager::VT StatsManager::readValue(StatsHolder& stats,
         case StatsMethod::COUNT:
             return stats.count(level);
         case StatsMethod::AVG:
-            return stats.avg(level);
+            return stats.template avg<VT>(level);
```
Comment (dangleptr) [inline src/common/stats/StatsManager.inl]: Why the changes?

Comment (dutor) [reply src/common/stats/StatsManager.inl]: The default type argument here is `double` while `VT` is of type `int64_t`, thus introducing two meaningless type conversions.
